src/sfctl/custom_app.py

```diff
@@ -336,6 +336,7 @@ def upgrade(  # pylint: disable=too-many-arguments,too-many-locals,missing-docst
         client, application_id, application_version, parameters,
         mode="UnmonitoredAuto", replica_set_check_timeout=None,
         force_restart=None, failure_action=None,
+        sort_order="Default",
```

Comment:
So this is really a deceptive argument name since it makes it sound like you're sorting something rather than adjusting the mechanism by which you progress through UDs, I would recommend naming it `traversal_order` or something like `UD_walk_order`

Reply:
Changing it here would mean it wouldn't be called the same it is called in other places (C#, C++ API, PowerShell) etc. I think it is too late to change that now (it is shipping in 6.5). Do you think it's best to keep it consistent, or change it here?